.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,16 @@
+# This workflow will launch at 09:00 every Monday
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
+
+jobs:
+  scala-steward:
+    runs-on: ubuntu-latest
+    name: Launch Scala Steward
+    steps:
+      - name: Launch Scala Steward
+        uses: scala-steward-org/scala-steward-action@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          sign-commits: true
```

Comment:
```suggestion
          sign-commits: true
          scala-steward-version: 0.8.1
```

Reply:
Why do you want to fix the version?

Reply:
Because the default is old. But whatever, I guess it doesn't matter.